libs/utils/src/environment_variable.ts

```diff
@@ -38,9 +38,6 @@ export enum BooleanEnvironmentVariableName {
   SKIP_CAST_VOTE_RECORDS_AUTHENTICATION = 'REACT_APP_VX_SKIP_CAST_VOTE_RECORDS_AUTHENTICATION',
   // Disables exporting original snapshots with CVRs
   DISABLE_CVR_ORIGINAL_SNAPSHOTS = 'REACT_APP_VX_DISABLE_CVR_ORIGINAL_SNAPSHOTS',
-  // Enables continuous export to USB on VxScan, usage of the corresponding export format on
-  // VxCentralScan, and handling of these exports on VxAdmin
-  ENABLE_CONTINUOUS_EXPORT = 'REACT_APP_VX_ENABLE_CONTINUOUS_EXPORT',
```

Comment:
Does this need to be removed from the root `.env` file too?

Reply:
Looks like I never added it to that file (oops), so we should be good to go:

```
$ cd vxsuite
$ git grep ENABLE_CONTINUOUS_EXPORT
$
```